src/Clapi/Types/Digests.hs

```diff
@@ -70,6 +72,9 @@ data TrpDigest = TrpDigest
   , trpdDefinitions :: Map (Tagged Definition Seg) (DefOp Definition)
   , trpdData :: DataDigest
   , trpdContOps :: ContOps Seg
+  -- FIXME: should errors come in a different digest to data updates? At the
```

Comment:
They were here because we were saying that the errors could be the cause of changes (and that putting them in a bundle could imply that).
So I don't know on point 1, the second half of this fixme makes sense to me.

Reply:
I think you had to look at the second half of this already when you fixed the tests, so perhaps delete the second half of this FIXME?